src/collection/dimensions.js

```diff
@@ -266,6 +266,33 @@ fn = elesfn = ({
         };
       }
 
+      function computePaddingValues( width, height, paddingObject, relativeTo ) {
+        // Assuming percentage is number from 0 to 1
```

Comment:
`.pstyle()` / `.parsedStyle()` gives the parsed style property

`.pstyle().pfValue` is the value in preferred units that can be used for calculations easily

`.pstyle().value` is the value as specified in the stylesheet (e.g. `20` for `20%`)